HOW-TO-EDITORIAL.md

```diff
@@ -0,0 +1,752 @@
+| `components.hero.kangaroo.eventInfo.items` | Solo per eventi | Dati su luogo, partecipazione, data/orario |
+
+---
+
+### Schemi YAML completi (puoi fare copia/incolla)
+- [Schema file Notizie](#schema-file-yaml-specifico-per-notizie)  
+- [Schema file Eventi](#schema-file-yaml-specifico-per-eventi)  
+- [Schema file Media](#schema-file-yaml-specifico-per-media)  
+
+---
+
+## Processo di Pubblicazione
+
+### 1. Preparazione dei File
+
+1. **Crea il file YAML** seguendo lo schema appropriato  
+2. **Nomina il file** con formato `YYYYMM-titolo-tutto-minuscolo.yaml` (corrisponde allo slug)  
+3. **Prepara immagini ottimizzate:**  
+   - Formato: JPG o PNG  
+   - Peso massimo: 800Kb  
+   - Dimensioni consigliate:  
+     - Hero/cover: **1920×923 px**  
+     - SEO/social: **1200×630 px** 
+4. **Recupera il link al video su YouTube (video)**
+5. **Prepara eventuali sottotitoli (video):**  
+   - Formato: WebVTT (`.vtt`)  
+   - Percorso: `/static/files/community/subtitles/`  
+6. **Prepara eventuali trascrizioni (video/podcast):**  
+   - Formato: OpenDocument (`.odt`)  
+   - Percorso: `/static/files/community/subtitles/`
+
+#### Convenzioni nomi file:
+
+##### File YAML
+- Formato: `YYYYMM-slug-tutto-minuscolo.yaml`
+- Data: quella di pubblicazione prevista, non di scrittura
+- Slug: solo lettere (non accentate), numeri, trattini
+
+##### Immagini e file correlati
+- Hero: `YYYYMM-slug-1920x923.jpg` (o dimensione reale in px)
+- SEO: `YYYYMM-slug-1200x630.jpg` (o dimensione reale in px)
+- Inline: `YYYYMM-slug-descrizione-immagine.jpg`
+- Sottotitoli: `YYYYMM-slug-contenuto.vtt`
+- Trascrizioni: `YYYYMM-slug-contenuto.odt`
+
+---
+
+### 2. Creazione della Pull Request
+1. **Fork** del repository `italia/designers.italia.it`  
+2. **Crea un branch** con nome descrittivo, es. `content/notizia-assistente-virtuale`  
+3. **Aggiungi i file nei percorsi corretti:**  
+   ```
+   src/data/content/community/[tipo]/[nome-file].yaml
+   static/images/community/[tipo]/
+   static/files/community/subtitles/[file-sottotitoli|trascrizioni] (se necessari)
+   ```
+4. **Commit** con messaggio chiaro:
+   ```
+   content(feat): aggiungi notizia su assistenti virtuali
+   
+   - Aggiunto file YAML notizia
+   - Caricate immagini hero e seo
+   - Verificati link e metadati SEO
+   ```
+5. **Apri la PR** verso `main`  
+6. **Controlla l’anteprima Vercel** collegata alla PR su GitHub (alcuni minuti per generarla, fai attenzione se ci fossero errori di build)
+7. **Controlla i test automatici** associati alla PR
+8. **Marca la PR come "draft"** finché ci lavori
+9. **Chiedi la "review"** alla redazione e/o maintainer
+
+Nota bene: **crea la PR almeno una settimana/dieci giorni prima della data di pubblicazione prevista**, per garantire tempo sufficiente per review, eventuali correzioni e rilascio in produzione.
+
+
+---
+
+### 3. Checklist di review
```

Comment:
Questa checklist sarebbe molto utile come template di PR dedicata alla pubblicazione di nuovi contenuti, altrimenti qui dentro rischia poi di essere dimenticata durante il processo di pubblicazione.

Reply:
Potrebbe essere una buona idea fare un template PR per la gestione editoriale. Vuoi provare a comporlo in una PR dedicata?

Reply:
Potremmo fare un template anche per le PR della guida rilasci DS.

Reply:
Intanto mergio questa, poi faremo i template PR con calma.